.azure/pipelines/azure-pipelines-mirror-within-azdo.yml

```diff
@@ -0,0 +1,61 @@
+# Configure which branches trigger builds
+trigger:
+  batch: true
+  branches:
+    include:
+    - internal/release/6.0
+
+variables:
+  - group: Mirror-Credentials
+
+# Merges code from one AzDO branch into another
+jobs:
+- template: /eng/common/templates/jobs/jobs.yml
+  parameters:
+    enableTelemetry: true
+    helixRepo: dotnet/arcade
+    jobs:
+    - job: Merge_Azure_DevOps_Branches
+      enableSBOM: false
+      condition: eq(variables['Build.Reason'], 'IndividualCI')
```

Comment:
I doubt this will ever be `true`. Suggest making the `internal` bit explicit (even though no `internal/release/*` branches exist in public) and matching `BUILD_REASON = BatchedCI` as seen in all rolling and official builds.

Reply:
Whoops, good catch on batched vs. individual - fixed (and added the `internal` condition as well